Sources/MountebankSwift/Api/Mountebank.swift

```diff
@@ -76,12 +78,18 @@ public struct Mountebank {
     /// basic information and hypermedia. If you want more information, either get the single ``Imposter`` or use the
     /// `replayable` flag.
     ///
+    /// - Parameters:
+    ///   - parameters: The parameters for changing the imposter response
+    ///
     /// - Returns: A list of all ``Imposters``
     ///
     /// - Throws: `MountebankValidationError` if connection between the client and server fails in some way.
     @discardableResult
     public func getAllImposters() async throws -> Imposters {
```

Comment:
```suggestion
    public func getAllImposters() async throws -> [Imposter] {
``` 

Perhaps just return an array here?

Reply:
Good point, will make the Imposters struct internal;